pkg/ucp/ucpclient/ucpclient.go

```diff
@@ -0,0 +1,146 @@
+/*
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ucpclient
+
+import (
+	"context"
+	"fmt"
+
+	ucpv20231001 "github.com/radius-project/radius/pkg/ucp/api/v20231001preview"
+)
+
```

Comment:
Is this a new client abstraction? My hope was that we could use the generated clients directly and avoid defining a new abstraction.

Reply:
removed